### PR TITLE
Updated docs for many VisualServer methods

### PR DESCRIPTION
**Pull request by clayjohn**
The first of a few PRs to flesh out the docs. This is all I could get through in the afternoon.

### CODE REVIEW

File: doc/classes/VisualServer.xml [hunk 1]
```diff
@@ -1674,6 +1720,7 @@
 			<argument index="1" name="tex_uv" type="Vector2">
 			</argument>
 			<description>
+				Sets the uv to be used with next vertex. Equivalent to [method ImmediateGeometry.set_uv].
```
Comment (Calinou) [inline doc/classes/VisualServer.xml]: Should we write `UV` instead of `uv` here?

File: doc/classes/VisualServer.xml [hunk 2]
```diff
@@ -1684,6 +1731,7 @@
 			<argument index="1" name="tex_uv" type="Vector2">
 			</argument>
 			<description>
+				Sets the uv2 to be used with next vertex. Equivalent to [method ImmediateGeometry.set_uv2].
```
Comment (Calinou) [inline doc/classes/VisualServer.xml]: Same as above.

File: doc/classes/VisualServer.xml [hunk 3]
```diff
@@ -80,6 +83,7 @@
 			<argument index="4" name="z_far" type="float">
 			</argument>
 			<description>
+				Set camera to use frustom projection. This mode allows adjusting the "offset" argument to create "tilted frustum" effects.
```
Comment (akien-mga) [inline doc/classes/VisualServer.xml]: Typo, frust**u**m.
Also for arguments, we usually use `[code]` markup when referring to their identifier.

File: doc/classes/VisualServer.xml [hunk 4]
```diff
@@ -128,6 +135,7 @@
 			<argument index="1" name="enable" type="bool">
 			</argument>
 			<description>
+				If true, preserves the horizontal aspect ratio which is equivalent to [constant Camera.KEEP_WIDTH]. If false, preserves the vertical aspect ratio which is equivalent to [constant Camera.KEEP_HEIGHT].
```
Comment (akien-mga) [inline doc/classes/VisualServer.xml]: Use code markup around true/false.

File: doc/classes/VisualServer.xml (hunk 3 above)
Comment (akien-mga) [reply doc/classes/VisualServer.xml]: Should also be "Sets" instead of "Set" (same for other methods below).

File: doc/classes/VisualServer.xml [hunk 5]
```diff
@@ -991,12 +1007,14 @@
 			<argument index="1" name="frame_step" type="float" default="0.0">
 			</argument>
 			<description>
+				Draws a frame. *This method is deprecated*, please use [method force_draw] instead.
```
Comment (akien-mga) [inline doc/classes/VisualServer.xml]: If you want italic markup, use `[i][/i]` :)

File: doc/classes/VisualServer.xml [hunk 6]
```diff
@@ -909,6 +923,7 @@
 			<argument index="2" name="max_z" type="int">
 			</argument>
 			<description>
+				Sets the z range of objects that will be affected by this light. Equivalent to [member Light2D.range_z_min] and [member Light2D.range_z_max].
```
Comment (akien-mga) [inline doc/classes/VisualServer.xml]: We currently tend to use capital Z when referring to the Z component in the docs. It's not fully consistent though, might need harmonization:
```
$ rg "[ ]Z[ ,]" | wc -l
53
$ rg "[ ]z[ ,]" | wc -l
5
```

File: doc/classes/VisualServer.xml [hunk 7]
```diff
@@ -1039,6 +1059,7 @@
 			<argument index="1" name="bg" type="int" enum="VisualServer.EnvironmentBG">
 			</argument>
 			<description>
+				Sets the *BGMode* of the environment. Equivalent to [member Environment.background_mode].
```
Comment (akien-mga) [inline doc/classes/VisualServer.xml]: Same here, use `[i]` if you meant to have italics markup.

File: doc/classes/VisualServer.xml [hunk 8]
```diff
@@ -1087,6 +1111,7 @@
 			<argument index="5" name="quality" type="int" enum="VisualServer.EnvironmentDOFBlurQuality">
 			</argument>
 			<description>
+
```
Comment (akien-mga) [inline doc/classes/VisualServer.xml]: Missing.

File: doc/classes/VisualServer.xml [hunk 9]
```diff
@@ -1105,6 +1130,7 @@
 			<argument index="5" name="quality" type="int" enum="VisualServer.EnvironmentDOFBlurQuality">
 			</argument>
 			<description>
+				Sets the values to be used with the "dof near blur" post-process effect. See [Environment] for more details.
```
Comment (akien-mga) [inline doc/classes/VisualServer.xml]: It appears in the inspector as "Dof Near Blur", so I think we should probably capitalize it similarly here (same for "adjustments" above). Might also remove the need for quotes.

Comment (akien-mga) [reply doc/classes/VisualServer.xml]: Alternatively, keep it to natural language and capitalize DoF as it should be.

File: doc/classes/VisualServer.xml [hunk 10]
```diff
@@ -1195,6 +1225,7 @@
 			<argument index="1" name="sky" type="RID">
 			</argument>
 			<description>
+				Sets the [Sky] to be used as the environment's background when using *BGMode* sky. Equivalent to [member Environment.background_sky].
```
Comment (akien-mga) [inline doc/classes/VisualServer.xml]: Italics.

File: doc/classes/VisualServer.xml [hunk 11]
```diff
@@ -1205,6 +1236,7 @@
 			<argument index="1" name="scale" type="float">
 			</argument>
 			<description>
+				Sets a custom fov for the background [Sky]. Equivalent to [member Environment.background_sky_custom_fov].
```
Comment (akien-mga) [inline doc/classes/VisualServer.xml]: Capitalize FOV, and maybe define it (unless it's done in the Environment method already).

File: doc/classes/VisualServer.xml [hunk 12]
```diff
@@ -1215,6 +1247,7 @@
 			<argument index="1" name="orientation" type="Basis">
 			</argument>
 			<description>
+				Sets the rotation of the backgroun [Sky] expressed as a [Basis]. Equivalent to [member Environment.background_sky_orientation].
```
Comment (akien-mga) [inline doc/classes/VisualServer.xml]: Typo, backgroun**d**

File: doc/classes/VisualServer.xml [hunk 13]
```diff
@@ -1247,6 +1280,7 @@
 			<argument index="12" name="bilateral_sharpness" type="float">
 			</argument>
 			<description>
+				Sets the variables to be used with the "screen space ambient occlusion" post-process effect. See [Environment] for more details.
```
Comment (akien-mga) [inline doc/classes/VisualServer.xml]: Same as above, I think "with the Screen Space Ambient Occlusion (SSAO) post-process effect" would read better.

**Comment by akien-mga** [comment]
Thanks a lot!

**Pull request merged**